### DISCUSSION TITLE
Reactflow toolbar and rc-slider

### DISCUSSION THREAD

**Discussion by amintabrizi** [xyflow/xyflow#2742] [General]
I want to use range input in React flow toolbar. For this feature, I am using Rc-slider but unfortunately it does not work in React flow toolbar,I guess the problem is drag event, Is it true?

https://codesandbox.io/s/cool-currying-7ggn6q?file=/src/CustomNode.js

**Comment by moklick** [comment]
For this you need to use the class name "nopan" instead of "nodrag" https://codesandbox.io/s/inspiring-mcnulty-fgsxfu

**Reply by amintabrizi** [reply]
It works great now, I couldn't find this class in the docs, if there is a link can you share it?
